README.md

```diff
@@ -122,6 +122,7 @@ As this template is strongly opinionated, the following provides a curated list
 - [vitailse](https://github.com/zynth17/vitailse) by [@zynth17](https://github.com/zynth17) - Like Vitesse but with TailwindCSS
 - [vitesse-modernized-chrome-ext](https://github.com/xiaoluoboding/vitesse-modernized-chrome-ext) by [@xiaoluoboding](https://github.com/xiaoluoboding) - ⚡️ Modernized Chrome Extension Manifest V3 Vite Starter Template
 - [vitesse-stackter-clean-architect](https://github.com/shamscorner/vitesse-stackter-clean-architect) by [@shamscorner](https://github.com/shamscorner) - A modular clean architecture pattern in vitesse template
+- [vitesse-enterprise](https://github.com/FranciscoKloganB/vitesse-enterprise) by [@FranciscoKloganB](https://github.com/FranciscoKloganB) - It combines `vitesse-stackter` with strict styling guides and component testing, enriched with pre-commit and pre-push hooks.
```

Comment:
Can you make the description a bit more concise? I think you can express the overall image of your template more. Styling guide and git hooks might sounds too detailed.

Reply:
Hey antfu. 🙇🏻 
How does this look?